backend/app/assets/stylesheets/spree/backend/components/_icons.scss

```diff
@@ -1,257 +1,219 @@
-  cloud-upload
-  cog
-  collapse-down
-  collapse-up
-  comment
-  compressed
-  copyright-mark
-  credit-card
-  cutlery
-  dashboard
-  download
-  download-alt
-  earphone
-  eject
-  envelope
-  euro
-  exclamation-sign
-  expand
-  export
-  eye-close
-  eye-open
-  facetime-video
-  fast-backward
-  fast-forward
-  file
-  film
-  filter
-  fire
-  flag
-  flash
-  floppy-disk
-  floppy-open
-  floppy-remove
-  floppy-save
-  floppy-saved
-  folder-close
-  folder-open
-  font
-  forward
-  fullscreen
-  gbp
-  gift
-  glass
-  globe
-  hand-down
-  hand-left
-  hand-right
-  hand-up
-  hd-video
-  hdd
-  header
-  headphones
-  heart
-  heart-empty
-  home
-  import
-  inbox
-  indent-left
-  indent-right
-  info-sign
-  italic
-  leaf
-  link
-  list
-  list-alt
-  lock
-  log-in
-  log-out
-  magnet
-  map-marker
-  minus
-  minus-sign
-  move
-  music
-  new-window
-  off
-  ok
-  ok-circle
-  ok-sign
-  open
-  paperclip
-  pause
-  pencil
-  phone
-  phone-alt
-  picture
-  plane
-  play
-  play-circle
-  plus
-  plus-sign
-  print
-  pushpin
-  qrcode
-  question-sign
-  random
-  record
-  refresh
-  registration-mark
-  remove
-  remove-circle
-  remove-sign
-  repeat
-  resize-full
-  resize-horizontal
-  resize-small
-  resize-vertical
-  retweet
-  road
-  saved
-  screenshot
-  sd-video
-  search
-  send
-  share
-  share-alt
-  shopping-cart
-  signal
-  sort
-  sort-by-alphabet
-  sort-by-alphabet-alt
-  sort-by-attributes
-  sort-by-attributes-alt
-  sort-by-order
-  sort-by-order-alt
-  sound-5-1
-  sound-6-1
-  sound-7-1
-  sound-dolby
-  sound-stereo
-  star
-  star-empty
-  stats
-  step-backward
-  step-forward
-  stop
-  subtitles
-  tag
-  tags
-  tasks
-  text-height
-  text-width
-  th
-  th-large
-  th-list
-  thumbs-down
-  thumbs-up
-  time
-  tint
-  tower
-  transfer
-  trash
-  tree-conifer
-  tree-deciduous
-  unchecked
-  upload
-  usd
-  user
-  volume-down
-  volume-off
-  volume-up
-  warning-sign
-  wrench
-  zoom-in
-  zoom-out
-);
+.icon-sm {
+  width: 0.8rem !important;
+  height: 0.8rem !important;
+}
 
-// We prefer alias/remap these so its easy to understand which icon to use.
-// Note: If desired alias exist as default, then remove it from default Array,
-//       or write a smarter @each scope below that can actually override.
-// ==========> default | alias
-$alias_icons: (plus add)
-              (pencil edit)
-              (ok capture)
-              (remove void)
-              (bullhorn promotion)
-              (share return)
-              (send shipment)
-              (share clone)
-              (refresh update)
-              (remove cancel)
-              (resize-full split)
-              (ok save)
-              (eye-open show)
-              (remove delete)
-              (minus refund)
-              (th-large products)
-              (th-large variants)
-              (picture images)
-              (tag properties)
-              (inbox stock)
-              (usd money)
```

Comment:
Please update also https://github.com/spree/spree/blob/master/backend/app/views/spree/admin/shared/_product_tabs.html.erb#L49 with correct icon.

Reply:
Hi @kaliszKrzysztof, Maybe PR #10279  would be a better way working? As used in the frontend with Inline SVG?

Those CSS versions in this PR rasterise on color change using filter, and can't be styled for path color.